.env.example

```
@@ -13,16 +13,23 @@ JWT_SECRET=your-super-secret-jwt-key-change-in-production-min-32-chars
 JWT_ACCESS_EXPIRES=15m
 JWT_REFRESH_EXPIRES=7d
 
-# AI Integration (Gemini — https://aistudio.google.com/apikey)
+# Provider selection
+# Use comma-separated list for automatic fallback (e.g. AI_PROVIDER=gemini,openai)
+STORAGE_PROVIDER=supabase            # supabase | s3 | r2
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)